src/lib.rs

```diff
@@ -0,0 +1,5 @@
+mod framerate;
+mod framerate_test;
```

Comment:
I would #[cfg(test)] this, otherwise users will see `framerate_test` as a path they can use in most ides